scalgrafanalib/layout.py

```diff
@@ -0,0 +1,90 @@
+from operator import floordiv
+from typing import List, Union
+import attr
+from grafanalib.core import GridPos, Panel, RowPanel  # type: ignore
+
+PanelList = List[Panel]
+PanelRow = PanelList
+PanelColumn = List[Union[Panel, PanelRow]]
+
+
+def get_height(panel: Panel) -> int:
+    """Return the height of the panel"""
+    if isinstance(panel, RowPanel):
+        return 1
+    if panel.gridPos is None:
+        return 0
+    return panel.gridPos.h
+
+
+def get_width(panel: Panel) -> int:
+    """Return the width of the panel"""
+    if isinstance(panel, RowPanel):
+        return 24
```

Comment:
As much as `1` and `0` sound like sane defaults, we don't know where this `24` comes from. Might be worth setting all 3 as constants (i.e. defined at top-level, all caps)